config/config.go

```diff
@@ -674,6 +682,15 @@ type QueueConfig struct {
 	MaxBackoff model.Duration `yaml:"max_backoff,omitempty"`
 }
 
+// MetadataConfig is the configuration for sending metadata to remote
+// storage.
+type MetadataConfig struct {
```

Comment:
Nothing should be configurable here, as I've previously mentioned. Among other things this is a temporary hack, anything we add here will almost certainly be replaced in the near future so it'd just cause unnecessary churn for users.

Hardcode to on and choose an internval.